### PR TITLE
Add Python 3 support to windowSplitter

### PR DESCRIPTION
**Pull request by nsoranzo**
Also:
- Update `bx-python` dependency
- Remove `tool_dependencies.xml`

### CODE REVIEW

File: tools/windowsplitter/windowSplitter.py [hunk 1]
```diff
@@ -1,94 +1,92 @@
...
+    fo = open(out_file, 'w')
 
     skipped_lines = 0
     first_invalid_line = 0
     invalid_line = None
     if offset == 0:
         makesliding = 0
 
-    for i, line in enumerate( file( inp_file ) ):
+    for i, line in enumerate(open(inp_file)):
```
Comment (bernt-matthias) [inline tools/windowsplitter/windowSplitter.py:53]: One could use a `with` here

**Comment by bernt-matthias** [comment]
Anything left from your side? Otherwise I will merge.

**Comment by bernt-matthias** [comment]
Deployment did not work. I guess the `SHED_KEY` has not been added to the secrets of this repo. Can you add this?

**Comment by nsoranzo** [comment]
I don't have the privileges (or the `devteam` TS passwords) unfortunately.

@martenson @davebx Can you help, maybe? We need the `TS_API_KEY` and `TTS_API_KEY` secrets to be set at https://github.com/galaxyproject/tools-devteam/settings/secrets .

**Comment by mvdbeek** [comment]
I have the key but can't access the secrets of this repo :/

**Comment by martenson** [comment]
I have set both TS_API_KEY and TTS_API_KEY

**Comment by nsoranzo** [comment]
@martenson Thanks, re-running the CI build for the merge commit uploaded successfully to the TS and TTS.

**Comment by bernt-matthias** [comment]
Thanks. Deployment worked

**Pull request merged**